src/arrays.jl

```diff
@@ -0,0 +1,388 @@
+using SymbolicUtils
+using StaticArrays
```

Comment:
Needs `StaticArrays` and its compat in Project.toml.

Reply:
Fixed